unify-front-end/components/sanity/RichTextRenderer.tsx

```
@@ -409,8 +409,17 @@ export default function RichTextRenderer({
           );
         case 'normal':
         default:
+          // Merged blocks (with \n) already handle spacing via lineHeight
+          // Non-merged blocks (paragraph breaks) get normal marginBottom
+          const hasLineBreaks = block.children?.some((child: any) => 
+            child.text === '\n'
+          );
+          const blockStyle = hasLineBreaks
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)